diff --git a/Bridges/__pycache__/__init__.cpython-37.pyc b/Bridges/__pycache__/__init__.cpython-37.pyc
index 748c80a..6945bfa 100644
Binary files a/Bridges/__pycache__/__init__.cpython-37.pyc and b/Bridges/__pycache__/__init__.cpython-37.pyc differ
diff --git a/Bridges/__pycache__/bridge.cpython-37.pyc b/Bridges/__pycache__/bridge.cpython-37.pyc
index d31942c..163ceb3 100644
Binary files a/Bridges/__pycache__/bridge.cpython-37.pyc and b/Bridges/__pycache__/bridge.cpython-37.pyc differ
diff --git a/Bridges/__pycache__/carla_bridge.cpython-37.pyc b/Bridges/__pycache__/carla_bridge.cpython-37.pyc
index 2f8c9c7..c27b47d 100644
Binary files a/Bridges/__pycache__/carla_bridge.cpython-37.pyc and b/Bridges/__pycache__/carla_bridge.cpython-37.pyc differ
diff --git a/ROAR/__pycache__/__init__.cpython-37.pyc b/ROAR/__pycache__/__init__.cpython-37.pyc
index 455e188..89d68d0 100644
Binary files a/ROAR/__pycache__/__init__.cpython-37.pyc and b/ROAR/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/agent_module/__pycache__/__init__.cpython-37.pyc b/ROAR/agent_module/__pycache__/__init__.cpython-37.pyc
index 84e3be8..8a87e2e 100644
Binary files a/ROAR/agent_module/__pycache__/__init__.cpython-37.pyc and b/ROAR/agent_module/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/agent_module/__pycache__/agent.cpython-37.pyc b/ROAR/agent_module/__pycache__/agent.cpython-37.pyc
index 0b1ba9f..49b6a35 100644
Binary files a/ROAR/agent_module/__pycache__/agent.cpython-37.pyc and b/ROAR/agent_module/__pycache__/agent.cpython-37.pyc differ
diff --git a/ROAR/agent_module/__pycache__/pure_pursuit_agent.cpython-37.pyc b/ROAR/agent_module/__pycache__/pure_pursuit_agent.cpython-37.pyc
index c0ee652..b15ce97 100644
Binary files a/ROAR/agent_module/__pycache__/pure_pursuit_agent.cpython-37.pyc and b/ROAR/agent_module/__pycache__/pure_pursuit_agent.cpython-37.pyc differ
diff --git a/ROAR/configurations/__pycache__/configuration.cpython-37.pyc b/ROAR/configurations/__pycache__/configuration.cpython-37.pyc
index 9449546..0be15b5 100644
Binary files a/ROAR/configurations/__pycache__/configuration.cpython-37.pyc and b/ROAR/configurations/__pycache__/configuration.cpython-37.pyc differ
diff --git a/ROAR/control_module/__pycache__/__init__.cpython-37.pyc b/ROAR/control_module/__pycache__/__init__.cpython-37.pyc
index 82c261f..430719b 100644
Binary files a/ROAR/control_module/__pycache__/__init__.cpython-37.pyc and b/ROAR/control_module/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/control_module/__pycache__/controller.cpython-37.pyc b/ROAR/control_module/__pycache__/controller.cpython-37.pyc
index 6443e8f..4c185d0 100644
Binary files a/ROAR/control_module/__pycache__/controller.cpython-37.pyc and b/ROAR/control_module/__pycache__/controller.cpython-37.pyc differ
diff --git a/ROAR/control_module/__pycache__/pid_controller.cpython-37.pyc b/ROAR/control_module/__pycache__/pid_controller.cpython-37.pyc
index 391e7bc..d33e01a 100644
Binary files a/ROAR/control_module/__pycache__/pid_controller.cpython-37.pyc and b/ROAR/control_module/__pycache__/pid_controller.cpython-37.pyc differ
diff --git a/ROAR/control_module/__pycache__/pure_pursuit_control.cpython-37.pyc b/ROAR/control_module/__pycache__/pure_pursuit_control.cpython-37.pyc
index 0e2ca63..741d469 100644
Binary files a/ROAR/control_module/__pycache__/pure_pursuit_control.cpython-37.pyc and b/ROAR/control_module/__pycache__/pure_pursuit_control.cpython-37.pyc differ
diff --git a/ROAR/perception_module/__pycache__/__init__.cpython-37.pyc b/ROAR/perception_module/__pycache__/__init__.cpython-37.pyc
index c12f588..4b95d18 100644
Binary files a/ROAR/perception_module/__pycache__/__init__.cpython-37.pyc and b/ROAR/perception_module/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/perception_module/__pycache__/detector.cpython-37.pyc b/ROAR/perception_module/__pycache__/detector.cpython-37.pyc
index f6e7a5f..9b981ac 100644
Binary files a/ROAR/perception_module/__pycache__/detector.cpython-37.pyc and b/ROAR/perception_module/__pycache__/detector.cpython-37.pyc differ
diff --git a/ROAR/perception_module/__pycache__/obstacle_from_depth.cpython-37.pyc b/ROAR/perception_module/__pycache__/obstacle_from_depth.cpython-37.pyc
index d9da805..59ca4f5 100644
Binary files a/ROAR/perception_module/__pycache__/obstacle_from_depth.cpython-37.pyc and b/ROAR/perception_module/__pycache__/obstacle_from_depth.cpython-37.pyc differ
diff --git a/ROAR/planning_module/__pycache__/__init__.cpython-37.pyc b/ROAR/planning_module/__pycache__/__init__.cpython-37.pyc
index 62e6f0c..d12d9b1 100644
Binary files a/ROAR/planning_module/__pycache__/__init__.cpython-37.pyc and b/ROAR/planning_module/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/planning_module/__pycache__/abstract_planner.cpython-37.pyc b/ROAR/planning_module/__pycache__/abstract_planner.cpython-37.pyc
index dad5d0d..4359878 100644
Binary files a/ROAR/planning_module/__pycache__/abstract_planner.cpython-37.pyc and b/ROAR/planning_module/__pycache__/abstract_planner.cpython-37.pyc differ
diff --git a/ROAR/planning_module/behavior_planner/__pycache__/__init__.cpython-37.pyc b/ROAR/planning_module/behavior_planner/__pycache__/__init__.cpython-37.pyc
index 7bc0045..97e7e25 100644
Binary files a/ROAR/planning_module/behavior_planner/__pycache__/__init__.cpython-37.pyc and b/ROAR/planning_module/behavior_planner/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/planning_module/behavior_planner/__pycache__/behavior_planner.cpython-37.pyc b/ROAR/planning_module/behavior_planner/__pycache__/behavior_planner.cpython-37.pyc
index 174b209..e379bfb 100644
Binary files a/ROAR/planning_module/behavior_planner/__pycache__/behavior_planner.cpython-37.pyc and b/ROAR/planning_module/behavior_planner/__pycache__/behavior_planner.cpython-37.pyc differ
diff --git a/ROAR/planning_module/local_planner/__pycache__/__init__.cpython-37.pyc b/ROAR/planning_module/local_planner/__pycache__/__init__.cpython-37.pyc
index 19daef3..cc05083 100644
Binary files a/ROAR/planning_module/local_planner/__pycache__/__init__.cpython-37.pyc and b/ROAR/planning_module/local_planner/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/planning_module/local_planner/__pycache__/local_planner.cpython-37.pyc b/ROAR/planning_module/local_planner/__pycache__/local_planner.cpython-37.pyc
index 54bd8f0..28bf7ed 100644
Binary files a/ROAR/planning_module/local_planner/__pycache__/local_planner.cpython-37.pyc and b/ROAR/planning_module/local_planner/__pycache__/local_planner.cpython-37.pyc differ
diff --git a/ROAR/planning_module/local_planner/__pycache__/loop_simple_waypoint_following_local_planner.cpython-37.pyc b/ROAR/planning_module/local_planner/__pycache__/loop_simple_waypoint_following_local_planner.cpython-37.pyc
index 9b54d47..cc6755f 100644
Binary files a/ROAR/planning_module/local_planner/__pycache__/loop_simple_waypoint_following_local_planner.cpython-37.pyc and b/ROAR/planning_module/local_planner/__pycache__/loop_simple_waypoint_following_local_planner.cpython-37.pyc differ
diff --git a/ROAR/planning_module/local_planner/__pycache__/simple_waypoint_following_local_planner.cpython-37.pyc b/ROAR/planning_module/local_planner/__pycache__/simple_waypoint_following_local_planner.cpython-37.pyc
index f666feb..cb597cd 100644
Binary files a/ROAR/planning_module/local_planner/__pycache__/simple_waypoint_following_local_planner.cpython-37.pyc and b/ROAR/planning_module/local_planner/__pycache__/simple_waypoint_following_local_planner.cpython-37.pyc differ
diff --git a/ROAR/planning_module/mission_planner/__pycache__/__init__.cpython-37.pyc b/ROAR/planning_module/mission_planner/__pycache__/__init__.cpython-37.pyc
index e339030..101f2a2 100644
Binary files a/ROAR/planning_module/mission_planner/__pycache__/__init__.cpython-37.pyc and b/ROAR/planning_module/mission_planner/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/planning_module/mission_planner/__pycache__/mission_planner.cpython-37.pyc b/ROAR/planning_module/mission_planner/__pycache__/mission_planner.cpython-37.pyc
index 3cf685f..d33614d 100644
Binary files a/ROAR/planning_module/mission_planner/__pycache__/mission_planner.cpython-37.pyc and b/ROAR/planning_module/mission_planner/__pycache__/mission_planner.cpython-37.pyc differ
diff --git a/ROAR/planning_module/mission_planner/__pycache__/waypoint_following_mission_planner.cpython-37.pyc b/ROAR/planning_module/mission_planner/__pycache__/waypoint_following_mission_planner.cpython-37.pyc
index 0250514..54573e9 100644
Binary files a/ROAR/planning_module/mission_planner/__pycache__/waypoint_following_mission_planner.cpython-37.pyc and b/ROAR/planning_module/mission_planner/__pycache__/waypoint_following_mission_planner.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/__init__.cpython-37.pyc b/ROAR/utilities_module/__pycache__/__init__.cpython-37.pyc
index 00ab4de..e82ade8 100644
Binary files a/ROAR/utilities_module/__pycache__/__init__.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/camera_models.cpython-37.pyc b/ROAR/utilities_module/__pycache__/camera_models.cpython-37.pyc
index 86f93dc..221b205 100644
Binary files a/ROAR/utilities_module/__pycache__/camera_models.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/camera_models.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/data_structures_models.cpython-37.pyc b/ROAR/utilities_module/__pycache__/data_structures_models.cpython-37.pyc
index d636bc0..58a5a70 100644
Binary files a/ROAR/utilities_module/__pycache__/data_structures_models.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/data_structures_models.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/errors.cpython-37.pyc b/ROAR/utilities_module/__pycache__/errors.cpython-37.pyc
index d7e8cad..0c8dd30 100644
Binary files a/ROAR/utilities_module/__pycache__/errors.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/errors.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/module.cpython-37.pyc b/ROAR/utilities_module/__pycache__/module.cpython-37.pyc
index 1c8adb5..c6fc09d 100644
Binary files a/ROAR/utilities_module/__pycache__/module.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/module.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/occupancy_map.cpython-37.pyc b/ROAR/utilities_module/__pycache__/occupancy_map.cpython-37.pyc
index 6e2c1e3..5744478 100644
Binary files a/ROAR/utilities_module/__pycache__/occupancy_map.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/occupancy_map.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/utilities.cpython-37.pyc b/ROAR/utilities_module/__pycache__/utilities.cpython-37.pyc
index 7341979..dbdb981 100644
Binary files a/ROAR/utilities_module/__pycache__/utilities.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/utilities.cpython-37.pyc differ
diff --git a/ROAR/utilities_module/__pycache__/vehicle_models.cpython-37.pyc b/ROAR/utilities_module/__pycache__/vehicle_models.cpython-37.pyc
index 1152170..612c420 100644
Binary files a/ROAR/utilities_module/__pycache__/vehicle_models.cpython-37.pyc and b/ROAR/utilities_module/__pycache__/vehicle_models.cpython-37.pyc differ
diff --git a/ROAR_Sim/__pycache__/__init__.cpython-37.pyc b/ROAR_Sim/__pycache__/__init__.cpython-37.pyc
index 717c785..56330d4 100644
Binary files a/ROAR_Sim/__pycache__/__init__.cpython-37.pyc and b/ROAR_Sim/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/__pycache__/__init__.cpython-37.pyc b/ROAR_Sim/carla_client/__pycache__/__init__.cpython-37.pyc
index ad18fd4..5d4255f 100644
Binary files a/ROAR_Sim/carla_client/__pycache__/__init__.cpython-37.pyc and b/ROAR_Sim/carla_client/__pycache__/__init__.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/__pycache__/carla_runner.cpython-37.pyc b/ROAR_Sim/carla_client/__pycache__/carla_runner.cpython-37.pyc
index 34f7870..ec9ba16 100644
Binary files a/ROAR_Sim/carla_client/__pycache__/carla_runner.cpython-37.pyc and b/ROAR_Sim/carla_client/__pycache__/carla_runner.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/util/__pycache__/camera_manager.cpython-37.pyc b/ROAR_Sim/carla_client/util/__pycache__/camera_manager.cpython-37.pyc
index cab1782..565b192 100644
Binary files a/ROAR_Sim/carla_client/util/__pycache__/camera_manager.cpython-37.pyc and b/ROAR_Sim/carla_client/util/__pycache__/camera_manager.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/util/__pycache__/hud.cpython-37.pyc b/ROAR_Sim/carla_client/util/__pycache__/hud.cpython-37.pyc
index e5483e6..a515167 100644
Binary files a/ROAR_Sim/carla_client/util/__pycache__/hud.cpython-37.pyc and b/ROAR_Sim/carla_client/util/__pycache__/hud.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/util/__pycache__/keyboard_control.cpython-37.pyc b/ROAR_Sim/carla_client/util/__pycache__/keyboard_control.cpython-37.pyc
index 67f0c8a..4bae8a1 100644
Binary files a/ROAR_Sim/carla_client/util/__pycache__/keyboard_control.cpython-37.pyc and b/ROAR_Sim/carla_client/util/__pycache__/keyboard_control.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/util/__pycache__/sensors.cpython-37.pyc b/ROAR_Sim/carla_client/util/__pycache__/sensors.cpython-37.pyc
index a98bf88..f4a3fa9 100644
Binary files a/ROAR_Sim/carla_client/util/__pycache__/sensors.cpython-37.pyc and b/ROAR_Sim/carla_client/util/__pycache__/sensors.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/util/__pycache__/utilities.cpython-37.pyc b/ROAR_Sim/carla_client/util/__pycache__/utilities.cpython-37.pyc
index 15303f6..16733de 100644
Binary files a/ROAR_Sim/carla_client/util/__pycache__/utilities.cpython-37.pyc and b/ROAR_Sim/carla_client/util/__pycache__/utilities.cpython-37.pyc differ
diff --git a/ROAR_Sim/carla_client/util/__pycache__/world.cpython-37.pyc b/ROAR_Sim/carla_client/util/__pycache__/world.cpython-37.pyc
index a3db8ca..6e26563 100644
Binary files a/ROAR_Sim/carla_client/util/__pycache__/world.cpython-37.pyc and b/ROAR_Sim/carla_client/util/__pycache__/world.cpython-37.pyc differ
diff --git a/ROAR_Sim/configurations/__pycache__/configuration.cpython-37.pyc b/ROAR_Sim/configurations/__pycache__/configuration.cpython-37.pyc
index 683efc4..4bf0e9c 100644
Binary files a/ROAR_Sim/configurations/__pycache__/configuration.cpython-37.pyc and b/ROAR_Sim/configurations/__pycache__/configuration.cpython-37.pyc differ
diff --git a/ROAR_gym/configurations/wandb_configuration.json b/ROAR_gym/configurations/wandb_configuration.json
index 80dc65d..6083b9b 100644
--- a/ROAR_gym/configurations/wandb_configuration.json
+++ b/ROAR_gym/configurations/wandb_configuration.json
@@ -1 +1 @@
-{"run_id": "Run 5", "name": "", "project_name": "Full_Control", "entity": "roar"}
\ No newline at end of file
+{"run_id": "Run 5", "name": "", "project_name": "Yunhao_Minor_Map_Input_Change", "entity": "roar"}
\ No newline at end of file
diff --git a/ROAR_gym/e2eModel.py b/ROAR_gym/e2eModel.py
index f0a3788..0016e12 100644
--- a/ROAR_gym/e2eModel.py
+++ b/ROAR_gym/e2eModel.py
@@ -157,7 +157,7 @@ def wandb_run_init(wandb_hp_config, load=False, requested_run_id=None, use_rando
 
     # Create a wandb run variable
     wandb.tensorboard.patch(
-        tensorboardX=False,
+        tensorboard_x=False,
         pytorch=True,
     )
     run = wandb.init(
