diff --git a/ROAR_gym/configurations/ppo_configuration.py b/ROAR_gym/configurations/ppo_configuration.py
index 17b1310..df404a5 100644
--- a/ROAR_gym/configurations/ppo_configuration.py
+++ b/ROAR_gym/configurations/ppo_configuration.py
@@ -11,7 +11,7 @@ misc_params = {
   "env_name": 'roar-e2e-ppo-v0',
   "run_fps": 8,  # TODO Link to the environment RUN_FPS
   "model_directory": Path("./output/Yunhao_PPOe2e_Input_Change"),
-  "run_name": "Test Run 1",
+  "run_name": "CNN Modified Run 1",
   "total_timesteps": int(1e6),
 }
 
diff --git a/ROAR_gym/configurations/wandb_configuration.json b/ROAR_gym/configurations/wandb_configuration.json
index cfb2610..8a4106f 100644
--- a/ROAR_gym/configurations/wandb_configuration.json
+++ b/ROAR_gym/configurations/wandb_configuration.json
@@ -1 +1 @@
-{"run_id": "Test Run 1", "name": "", "project_name": "Yunhao_Minor_Map_Input_Change", "entity": "roar"}
\ No newline at end of file
+{"run_id": "CNN Modified Run 1", "name": "", "project_name": "Yunhao_Minor_Map_Input_Change", "entity": "roar"}
\ No newline at end of file
diff --git a/ROAR_gym/e2eModel.py b/ROAR_gym/e2eModel.py
index f0a3788..5d146a1 100644
--- a/ROAR_gym/e2eModel.py
+++ b/ROAR_gym/e2eModel.py
@@ -71,7 +71,7 @@ from stable_baselines3.common.vec_env import DummyVecEnv, VecVideoRecorder
 
 
 # imports for helper functions and torch cnn models
-from ppo_util import find_latest_model, CustomMaxPoolCNN, Atari_PPO_Adapted_CNN
+from ppo_util import find_latest_model, CustomMaxPoolCNN, Atari_PPO_Adapted_CNN, YunhaoModifiedAtariCNN
 
 
 
@@ -213,7 +213,7 @@ def main(pass_num):
     # Setting the feature extract or based on the environment mode
     if env.mode == 'baseline':
         policy_kwargs = dict(
-            features_extractor_class=Atari_PPO_Adapted_CNN,
+            features_extractor_class=YunhaoModifiedAtariCNN, #Atari_PPO_Adapted_CNN,
             features_extractor_kwargs=dict(features_dim=256)
         )
     else:
diff --git a/ROAR_gym/ppo_util.py b/ROAR_gym/ppo_util.py
index 6702b99..fb8e2d9 100644
--- a/ROAR_gym/ppo_util.py
+++ b/ROAR_gym/ppo_util.py
@@ -173,7 +173,6 @@ class CustomMaxPoolCNN_attention(BaseFeaturesExtractor):
         return self.fullStack(observations[0])
 
 
-
 class CustomMaxPoolCNN(BaseFeaturesExtractor):
     """
     the CNN network that interleaves convolution & maxpooling layers, used in a
@@ -424,6 +423,29 @@ class Atari_PPO_Adapted_CNN(BaseFeaturesExtractor):
         observations=observations.view(observations.shape[0],-1,*observations.shape[3:])
         return self.network(observations)
 
+class YunhaoModifiedAtariCNN(BaseFeaturesExtractor):
+    def __init__(self, observation_space: gym.spaces.Box, features_dim: int = 256):
+        super(Atari_PPO_Adapted_CNN, self).__init__(observation_space,features_dim)
+        channels = observation_space.shape[0]*observation_space.shape[1]
+        self.network = nn.Sequential(
+            # Scale(1/255),
+            layer_init(nn.Conv2d(channels, 32, 8, stride=4)),
+            nn.ReLU(),
+            layer_init(nn.Conv2d(32, 64, 4, stride=2)),
+            nn.ReLU(),
+            layer_init(nn.Conv2d(64, 64, 3, stride=1)),
+            nn.ReLU(),
+            layer_init(nn.Conv2d(64,16,1,stride=1)), #added
+            nn.ReLU(),
+            nn.Flatten(),
+            layer_init(nn.Linear(784, features_dim)), #shrinked
+            # nn.ReLU(),
+        )
+
+    def forward(self, observations: th.Tensor) -> th.Tensor:
+        observations=observations.view(observations.shape[0],-1,*observations.shape[3:])
+        return self.network(observations)
+
 def find_latest_model(root_path: Path) -> Optional[Path]:
     import os
     from pathlib import Path
